ccloud/docs/beginner-cloud.rst

```diff
@@ -433,11 +426,11 @@ Run a Java producer before and after configuring the ACLs
       ccloud kafka acl delete --allow --service-account 104349 --operation CREATE --topic demo-topic-1
       ccloud kafka acl delete --allow --service-account 104349 --operation WRITE --topic demo-topic-1
 
-   You should see two ``Deleted ACLs.`` messages.
+   Verify you see two ``Deleted ACLs.`` messages.
```

Comment:
Should this be in the ordered list now?

#. Verify you see two ``Deleted ACLs.`` messages.

Reply:
The tutorial is inconsistent.  Which do you prefer?  We can go with one consistently...

Reply:
Agreed. Let's go with "You should see..." or "Your output resembles..." I tried minimizing "verify" in spots where it was in steps back to back.